src/ray/core_worker/store_provider/memory_store_provider.cc

```diff
@@ -30,7 +30,8 @@ Status CoreWorkerMemoryStoreProvider::Put(const RayObject &object,
 Status CoreWorkerMemoryStoreProvider::Get(
     const std::unordered_set<ObjectID> &object_ids, int64_t timeout_ms,
     const TaskID &task_id,
-    std::unordered_map<ObjectID, std::shared_ptr<RayObject>> *results) {
+    std::unordered_map<ObjectID, std::shared_ptr<RayObject>> *results,
+    bool *got_exception) {
```

Comment:
It could, but that would require iterating over the full list of results even in success cases, which I would want to avoid especially because the `IsException` check loops over a list of types itself.